crates/subspace-farmer/src/farming.rs

```diff
@@ -45,12 +52,14 @@ impl Farming {
             receiver,
         ));
 
+        // return the instance
         Farming {
             sender: Some(sender),
             handle: Some(farming_handle),
         }
     }
 
+    /// takes the ownership of the handle, and simply waits on it
```

Comment:
Same here, you don't need to say that it takes ownership, it is obvious from function signature.